vcs/backends/p4/changeset.py

```diff
@@ -0,0 +1,253 @@
+import datetime
+
+from vcs.backends.base import BaseChangeset
+from vcs.utils.lazy import LazyProperty
+
+class P4Changeset(BaseChangeset):
+    """
+    P4 changelist. Submitted CLs will be implemented first.
+
+    **Attributes**
+
+        ``repository``
+            repository object within which changeset exists
+
+        ``id``
+            Changelist number (int)
+
+        ``raw_id``
+            same as id
+
+        ``short_id``
+            same as id
+
+        ``revision``
+            same as id
+
+        ``files``
+            list of ``FileNode`` (``Node`` with NodeKind.FILE) objects, TBD
+
+        ``dirs``
+            list of ``DirNode`` (``Node`` with NodeKind.DIR) objects, TBD
+
+        ``nodes``
+            combined list of ``Node`` objects, TBD
+
+        ``author``
+            author of the changeset, as unicode, TBD
+
+        ``message``
+            message of the changeset, as unicode
+
+        ``parents``
+            list of parent changesets, TBD
+
+        ``last``
+            ``True`` if this is last changeset in repository, ``False``
+            otherwise; trying to access this attribute while there is no
+            changesets would raise ``EmptyRepositoryError``, TBD
+
+        Added properties:
+
+        ``time``
```

Comment:
Do we have a standard field for this? This way it will not be possible to use my_changest.time regardless of my_changest's type (P4Changeset vs GitChangest vs HgChangeset)